wagtail/admin/templates/wagtailadmin/reports/base_report.html

```diff
@@ -4,32 +4,23 @@
 {% block titletag %}{{ title }}{% endblock %}
 
 {% block content %}
-    <header class="w-header" role="banner">
-        <div class="row">
-            <div class="left">
-                <div class="col">
-                    <h1 class="w-header__title">
-                        {% if header_icon %}{% icon class_name="w-header__glpyh" name=header_icon %}{% endif %}
-                        {{ title }}{% if subtitle %} <span class="w-header__subtitle">{{ subtitle }}</span>{% endif %}
-                    </h1>
-                </div>
-            </div>
-            <div class="right report__actions">
-                {% block actions %}
-                    {% if view.list_export %}
-                        <div class="dropdown dropdown-button match-width">
-                            <a href="{{ view.xlsx_export_url }}" class="button bicolor button--icon">{% icon name="download" wrapped=1 %}{% trans 'Download XLSX' %}</a>
-                            <div class="dropdown-toggle">{% icon name="arrow-down" %}</div>
-                            <ul>
-                                <li><a  class="button bicolor button--icon" href="{{ view.csv_export_url }}">{% icon name="download" wrapped=1 %}{% trans 'Download CSV' %}</a></li>
-                            </ul>
-                        </div>
-                    {% endif %}
-                {% endblock %}
-            </div>
+    {% fragment as report_actions %}
+        <div class="right report__actions">
```

Comment:
```suggestion
```

Let’s try to remove this `div` and rewrite the dropdown styles so we have a class directly on `dropdown` line 11. 

https://github.com/wagtail/wagtail/blob/main/client/scss/layouts/_report.scss#L50-L54